presto-rcfile/src/main/java/com/facebook/presto/rcfile/GenericRcFileDecompressor.java

```diff
@@ -21,12 +21,12 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class AircompressorDecompressor
+public class GenericRcFileDecompressor
```

Comment:
You can't call it `HadoopCompressor` because there is already one.

* `AircompressorCompressor` and `HadoopCompressor` are very similar. The only difference is whether `CodecPool` is used.

Maybe call them `Pooled` and `Unpooled`?